pages/index.js

```diff
@@ -1,109 +1,181 @@
+              />
+            </a>
+          </Link>
+          <div className="md:px-2 py-4 lg:p-6 md:max-w-[400px] lg:max-w-[600px]">
+            <Link href="/">
+              <a>
+                <h3 className="headline--3 link">
+                  Muzeum Ostře sledovaných vlaků
+                </h3>
+              </a>
+            </Link>
+            <div className="px-4 text-sm text-neutral-500">
+              <time dateTime="2022-03-15T13:25:30+01:00">15. 3. 2022</time>
+            </div>
+            <p className="paragraph">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. In
+              condimentum, arcu vel tempus finibus, nulla metus congue lacus, ac
+              vehicula ex nulla non erat. Phasellus ultrices sapien sit amet
+              eros ullamcorper lacinia. Sed at sem facilisis ipsum fermentum
+              pulvinar. Nam volutpat fringilla mi, vitae mollis mi lobortis
+              quis. Vivamus finibus ex est, interdum bibendum dolor consequat
+              in. Duis ut porta nunc. Vivamus tempor accumsan finibus.
+            </p>
 
-    <p className="">
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit. In condimentum,
-      arcu vel tempus finibus, nulla metus congue lacus, ac vehicula ex nulla
-      non erat. Phasellus ultrices sapien sit amet eros ullamcorper lacinia. Sed
-      at sem facilisis ipsum fermentum pulvinar. Nam volutpat fringilla mi,
-      vitae mollis mi lobortis quis. Vivamus finibus ex est, interdum bibendum
-      dolor consequat in. Duis ut porta nunc. Vivamus tempor accumsan finibus.
-    </p>
+            <ButtonLink href="/detail">Číst dál</ButtonLink>
+          </div>
+        </div>
+      </article>
 
-    <p className="">
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit. In condimentum,
-      arcu vel tempus finibus, nulla metus congue lacus, ac vehicula ex nulla
-      non erat. Phasellus ultrices sapien sit amet eros ullamcorper lacinia. Sed
-      at sem facilisis ipsum fermentum pulvinar. Nam volutpat fringilla mi,
-      vitae mollis mi lobortis quis. Vivamus finibus ex est, interdum bibendum
-      dolor consequat in. Duis ut porta nunc. Vivamus tempor accumsan finibus.
-    </p>
+      <h2 className="underline-medium text-center pt-12">Nejnovější články</h2>
 
-    <p className="mx-8 md:mx-20 lg:mx-36">
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit. In condimentum,
-      arcu vel tempus finibus, nulla metus congue lacus, ac vehicula ex nulla
-      non erat. Phasellus ultrices sapien sit amet eros ullamcorper lacinia. Sed
-      at sem facilisis ipsum fermentum pulvinar. Nam volutpat fringilla mi,
-      vitae mollis mi lobortis quis. Vivamus finibus ex est, interdum bibendum
-      dolor consequat in. Duis ut porta nunc. Vivamus tempor accumsan finibus.
-    </p>
+      <section className="md:flex w-full justify-center px-4">
+        <div className="md:flex w-full lg:w-[1088px] space-x-4">
```

Comment:
Maybe, if we want some max-width for content/images - we could define the same custom class for it in the tailwind config.